10/s2i/bin/assemble

```diff
@@ -78,19 +83,37 @@ if [ -z "$NODE_ENV" ]; then
   fi
 fi
 
-if [ "$NODE_ENV" != "production" ]; then
+echo -e "Current git config"
+git config --list
+
+if [ -n "$YARN_ENABLED" ]; then
```

Comment:
Implementation of `YARN_ENABLED` is fine. But we will have the same problem with NEXUS_REGISTRY.
I would prefer to add to the condition `NPM_MIRROR` also `yarn config set registry` or define a new variable called YARN_MIRROR.